packages/icons/components/Switch.tsx

```diff
@@ -0,0 +1,17 @@
+import type { FC } from 'react';
+import { Path, Svg } from 'react-native-svg';
+
+import type { IconProps } from './types';
+
+export const Switch: FC<IconProps> = ({ size = 24, color = '#FFFFFF' }) => {
+	return (
+		<Svg width={size} height={size} viewBox="0 0 17 19" fill="none">
```

Comment:
just my preference, when the `viewBox` is not a square (in this case, `17` in width and `19` in height or vice versa), I will throw it inside a square frame (which may be 19x19) in Figma and then expose that frame again to get a square icon xD

Reply:
:v I just keep the icon viewBox, it is not necessary to wrap it into a square box, width, and height do their job, viewBox is used for the relative size of child lines